### PR TITLE
app template added

### PR DESCRIPTION
**Pull request by Valcis**

### CODE REVIEW

**Review by agandia9 — COMMENTED** [review]
OK!

File: staff/pedrovalverde/js/challenges/scripts/countChars.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+count = countChars('abracadabra', function (c) { return c === 'a'; });
+
+console.log('countChars("abracadabra") should return 5', count === 5, count);
+
+
+try {
+    count = countChars('abracadabra', '...');
+} catch (err) {
+    console.log('countChars("abracadabra", "...") should throw an error', err !== undefined, err);
+}
```
Comment (Asix94) [inline staff/pedrovalverde/js/challenges/scripts/countChars.test.js]: use strict

File: staff/pedrovalverde/js/challenges/scripts/forEach.test.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+var a = [1,2,3];
+forEach(a, function (v) { console.log(v) });
```
Comment (Asix94) [inline staff/pedrovalverde/js/challenges/scripts/forEach.test.js:2]: use strict

File: staff/pedrovalverde/js/challenges/scripts/reduce.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+/*
+var o = { 
+    a: {value:1}, 
+    b: {value:2}, 
+    c: {value:3} 
+};
+
+Object.keys(o).reduce(function (previous, key) {
+    return previous + o[key].value;
+}, 0);*/
```
Comment (Asix94) [inline staff/pedrovalverde/js/challenges/scripts/reduce.test.js:31]: use strict

**Pull request merged**